server/helpers/validations/joiValidations.js

```diff
@@ -0,0 +1,15 @@
+import Joi from 'joi';
+
+const joiValidations = (input, joiSchema, joiOptions, res, next) => {
+  Joi.validate(input, joiSchema, joiOptions, (err) => {
+    if (err) {
+      // this regex replaces the string (\") that is returned with the json response
+      const getErrorMessages = error => error.message.replace(/"/g, '');
+      res.status(422).json({ message: err.details.map(getErrorMessages), status: 'failed', });
+    } else if (!err) {
+      return next();
+    }
+  });
+};
+
+export default joiValidations;
```

Comment:
This block of code is used by all the validations in the middlewares. So it has been moved to it s own file.